src/workflows-app/RunDetails.test.js

```diff
@@ -192,7 +192,7 @@ describe('BaseRunDetails - render smoke test', () => {
       // Checking row text content for dates since querying by formatted date doesn't seem to work
       const statusObj = collapseCromwellStatus(executionStatus, backendStatus);
       const status = within(row).getAllByText(statusObj.label());
-      expect(status.length).toEqual(2);
+      expect(status.length).toEqual(1);
```

Comment:
This value (and the ones below) changed from 2 to 1 because the tooltip was removed.